src/net_processing.cpp

```diff
@@ -678,7 +678,6 @@ class PeerManagerImpl final : public PeerManager
     std::shared_ptr<const CBlock> m_most_recent_block GUARDED_BY(m_most_recent_block_mutex);
```

Comment:
1aa6a5642327ee028d18b34e2c9b2be74c81454c commit message: s/is only to/is only used to/

?

Reply:
Updated commit text: s/that function is only to update/that function only updates/